### PR TITLE
RSA algorithm implemented in python in encryption_algorithms directory

### PR DESCRIPTION
**Pull request by Kuldip397**
PR under issue #10.

### CODE REVIEW

File: encryption_algorithms/RSA_algorithm.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    lastx = y = 1
+    while m != 0:
+        q = e // m
+        e, m = m, e % m
+        x, lastx = lastx - q*x, x
+        y, lasty = lasty - q*y, y
+    return lastx
+ 
+if __name__ == "__main__":
+    main()
```
Comment (t2013anurag) [inline encryption_algorithms/RSA_algorithm.py]: new line.

Comment (Kuldip397) [reply encryption_algorithms/RSA_algorithm.py]: Can you please elaborate what changes I'm supposed to make as I'm not getting what to with "new line".

Comment (Kuldip397) [reply encryption_algorithms/RSA_algorithm.py]: Done.

**Pull request merged**